docs/edgex-taf.rst

```diff
@@ -294,11 +284,10 @@ The usage for robot file is illustrated below::
     Documentation    Deploy EdgeX
     Library          TAF/testCaseModules/keywords/setup/setup_teardown.py
     Library          TAF/testCaseModules/keywords/setup/edgex.py
-    Library          TAF/testCaseModules/keywords/setup/consul.py
 
 The usage for python script is illustrated below::
 
-    TAF/utils/src/setup/edgex.py
+    TAF/testCaseModules/keywords/setup/edgex.py
 
     from TUC.data.SettingsInfo import SettingsInfo
     import TAF.utils.src.setup.startup_checker as checker
```

Comment:
Library startup_checker path should be updated.

Reply:
Updated.